src/main/java/core/basesyntax/JavaDateTimeApi.java

```diff
@@ -120,6 +168,9 @@ public OffsetDateTime offsetDateTime(LocalDateTime localTime) {
      * Example: "01 January 2000 18:00".
      */
     public String formatDate(LocalDateTime dateTime) {
-        return "";
+        DateTimeFormatter formatter = DateTimeFormatter
+                .ofPattern("dd MMMM yyyy HH:mm")
```

Comment:
the same as above

Reply:
totaly disagree. We use this pattern just in the method, why to do code fat?

Reply:
the same answer as above

Reply:
added explanation above